docs/userGuide/syntax/variables.mbdf

```diff
@@ -103,6 +103,73 @@ You can specify a default value for a variable, which is displayed when the vari
 
 Note: These variables will not be applied to [`<include>` files]({{ baseUrl }}/userGuide/reusingContents.html#the-include-tag). Additionally, global variables (`_markbind/variables.md`) will take precedence over any page variables. *See also: [Specifying Variables in an `<include>`]({{ baseUrl }}/userGuide/reusingContents.html#specifying-variables-in-an-include)*.
 
+### Importing Variables
+
+**You can access [page variables](#page-variables) from another page by importing them.**
+
+
+{{ icon_example }} Importing specific variables from `person.md` into `coverpage.md`:
+`person.md`:
+```html
+<variable name="address">123 Sun Avenue</variable>
+<variable name="name">Mark</variable>
+<variable name="phone">123456789</variable>
+```
+
+`coverpage.md`:
+
+```html
+<import address name from="page.md"/>
+
+```
+
+will allow you to access the variables as per normal: <code>{<span></span>{address}}</code>, <code>{<span></span>{name}}</code>, <code>{<span></span>{phone}}</code>.
```

Comment:
You could collapse this by directly using it in `coverpage.md`, saves some words.

Reply:
![image](https://user-images.githubusercontent.com/3646725/62019555-fe1c1a00-b1f1-11e9-9f2c-4232f98ef83f.png)

I think i'll keep it as it is, the change doesn't look very nice.